Source/Core/Plugins/EventProcessor/Default/03_ManualStackingPlugin.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Exceptionless.Core.Pipeline;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+
+namespace Exceptionless.Core.Plugins.EventProcessor {
+    [Priority(3)]
+    public class ManualStackingPlugin : EventProcessorPluginBase {
+        public override Task EventProcessingAsync(EventContext context) {
+            var data = context.Event.Data;
+            if (!data.ContainsKey(Event.KnownDataKeys.ManualStackingKey)
+                || data[Event.KnownDataKeys.ManualStackingKey] == null)
+                return Task.CompletedTask;
+
+            context.StackSignatureData.AddItemIfNotEmpty(nameof(Event.KnownDataKeys.ManualStackingKey), data[Event.KnownDataKeys.ManualStackingKey].ToString());
```

Comment:
It's the key name here wrong? I think it would resolve to "ManualStackingKey" instead of "@manual_stacking_key".